plugins/scaffolder/src/components/TemplateList/TemplateList.tsx

```diff
@@ -51,9 +52,18 @@ export const TemplateList = ({
 }: TemplateListProps) => {
   const { loading, error, entities } = useEntityList();
   const Card = TemplateCardComponent || TemplateCard;
-  const maybeFilteredEntities = group
-    ? entities.filter(e => group.filter(e))
-    : entities;
+  const featureFlagApi = useApi(featureFlagsApiRef);
```

Comment:
Still think that we want to make this configurable right? So we turn it on if you opt in right as @ahhhndre mentioned?